src/algorithms/rsa.rs

```diff
@@ -259,7 +259,7 @@ impl RSAKeyPairLike for RS256KeyPair {
     }
 
     fn padding_scheme(&self) -> rsa::PaddingScheme {
-        rsa::PaddingScheme::new_pkcs1v15_sign(Some(rsa::Hash::SHA2_256))
+        rsa::PaddingScheme::new_pkcs1v15_sign::<sha2::Sha256>()
```

Comment:
Does that work?

I'm a little bit scared by the fact that an object that depends on randomness doesn't get a RNG any more.

Maybe it's reusing the one from `sign_blinded()` but that would be very counterintuitive.

Before bumping the dependency, we need to make sure that the behavior stays the same and that it doesn't introduce a vulnerability.

Reply:
I had the same worry as you, mostly opened the PR to start the discussion, I fully agree with you that we need to be sure what the change was about before we merge this.

Reply:
Glancing at the code, it looks like the RNG is implicitly passed over from the signing operation to the PSS object. So , the behavior should be the same.